packages/server/utils/getPhase.ts

```diff
@@ -16,7 +16,7 @@ interface PhaseTypeLookup {
 }
 
 const getPhase = <T extends keyof PhaseTypeLookup>(phases: GenericMeetingPhase[], phaseType: T) => {
-  return phases.find((phase) => phase.phaseType === phaseType) as PhaseTypeLookup[T]
+  return phases.find((phase) => phase.phaseType === phaseType) as unknown as PhaseTypeLookup[T]
```

Comment:
some phases are now required to have at least 1 stage, while others can have 0 stages. since phases is a GenericMeetingPhase[], it can have 0 or more stages, but something like discuss or checkin in guaranteed to have at least 1. Since the type of stages mismatch (0+, 1+) we have to cast to unknown (or any) in order to cast it to something completely different